include/rmm/detail/export.hpp

```diff
@@ -24,4 +24,5 @@
 #else
 #define RMM_EXPORT
 #define RMM_HIDDEN
+#define RMM_NAMESPACE rmm
```

Comment:
This is necessary for anyone compiling with a compiler that doesn't advertise `__GNUC__`.